simulators/portal-interop/src/constants.rs

```diff
@@ -0,0 +1,29 @@
+// Execution Layer hard forks https://ethereum.org/en/history/
+pub const SHANGHAI_BLOCK_NUMBER: u64 = 17034870;
+pub const MERGE_BLOCK_NUMBER: u64 = 15537394;
+pub const LONDON_BLOCK_NUMBER: u64 = 12965000;
+pub const BERLIN_BLOCK_NUMBER: u64 = 12244000;
+pub const ISTANBUL_BLOCK_NUMBER: u64 = 9069000;
+pub const CONSTANTINOPLE_BLOCK_NUMBER: u64 = 7280000;
+pub const BYZANTIUM_BLOCK_NUMBER: u64 = 4370000;
+pub const HOMESTEAD_BLOCK_NUMBER: u64 = 1150000;
+
+// Tests can be written to be generate from this file.
+// Tests data will be in a yaml key value format
+// Tests consist of content key value pairs.
+// For history tests the header for the respective block body/receipt must be place before
+// so
+// block header 1
+// block body 1
+// block receipt 1
+// block header 2
+// block receipt 2
+// block header 3
+// would be a valid test but
+// block header 2
+// block receipt 3
+// block header 3
+// this wouldn't be a valid test file as block receipt 3 doesn't have a header place before it
+// related header, block body, receipts must be sequential
+// We want this so block forks with different formats can be tested, by updating an external test repo
+pub const TEST_DATA_FILE_PATH: &str = "./test-data/test_data_collection_of_forks_blocks.yaml";
```

Comment:
I'm guessing we will eventually add more test data to run these tests for other subnetworks. In which case, I'd imagine we'll have a separate test-file with that data (since it could need special treatment in how it's gossiped). Basically, I think a filename of `./test-data/history.yaml` is sufficient here. The current filename is kinda confusing. 

I like the improved description for how the test file should be configured, but I think that it would be better off at the top of the test file itself, since it's not obvious that future devs will see this page when they're adding tests.

I'd also add a comment listing exactly what tests will be generated & run on each content key/value pair that's added to the test data

Reply:
> I'm guessing we will eventually add more test data to run these tests for other subnetworks. In which case, I'd imagine we'll have a separate test-file with that data (since it could need special treatment in how it's gossiped). Basically, I think a filename of `./test-data/history.yaml` is sufficient here. The current filename is kinda confusing.

I think that is out of scope for this PR and should be done when support is added to write tests for other networks. I think the format should be ``./test-data/history/nameoftestdata.yaml`` This will match the formatting on the test data repo

> I like the improved description for how the test file should be configured, but I think that it would be better off at the top of the test file itself, since it's not obvious that future devs will see this page when they're adding tests.

There is already a comment in the test data file for this, so we can remove the comment above the constant if you want

> I'd also add a comment listing exactly what tests will be generated & run on each content key/value pair that's added to the test data

I will talk about this in discord dms cause I am confused about it

Reply:
> There is already a comment in the test data file for this

Sure, but from what I can tell it's missing all of the information that I suggested we add....
```
# Test data for Portal Hive
# Data is formatted by comment of block height, header, block body, and receipt, expect for blocks 1 and 100 which don't have receipts
```

So I think it's still worth moving this comment to that test file

Reply:
Oh Ok I think I understand now, I was confused earlier but it makes more sense now